packages/terra-table/src/TableSection.jsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import 'terra-base/lib/baseStyles';
+import SectionHeader from './TableSectionHeader';
+
+const propTypes = {
+  /**
+   * The children list items passed to the component.
+   */
+  children: PropTypes.node,
+  /**
+   * Number of columns the section header must span
+   */
+  colSpan: PropTypes.number.isRequired,
+  /**
+   * Whether or not the section is collapsed.
+   */
+  isCollapsed: PropTypes.bool,
+  /**
+   * Whether or not the section can be collapsed.
+   */
+  isCollapsible: PropTypes.bool,
+  /**
+   * Optionally sets the heading level. One of `1`, `2`, `3`, `4`, `5`, `6`.
+   */
+  level: PropTypes.oneOf([1, 2, 3, 4, 5, 6]),
```

Comment:
Should `1` be an option?  Will the table header need to use `1` and all table body content need 2 - 6?

Reply:
Updated.